tests/integration/facets.ts

```diff
@@ -121,7 +121,7 @@ it('facets are updated after a vertical search', async () => {
     query: { input: 'test query' },
     spellCheck: { enabled: true }
   };
-  const answers = createMockedHeadless(mockAnswersCore, initialState);
+  const answers = createMockedHeadless(mockSearchCore, initialState);
```

Comment:
I assume changing the other internal uses (like this `answers` variable name) is not part of this item?

Reply:
Since there are still some identifiers like `AnswersHeadless` and `AnswersCore` in the document that are still in the document, I thought I'd rename them along with other internal uses because the previous PRs mentioned that they are renamed.